Tophat/Models/AppleApplication.swift

```diff
@@ -68,8 +67,24 @@ struct AppleApplication: Application {
 	}
 
 	var platform: Platform {
-		// Only iOS supported, for now.
-		.iOS
+		guard let platformNames = bundle?.infoDictionary?["CFBundleSupportedPlatforms"] as? [String] else {
+			return .unknown
+		}
+
+		for platformName in platformNames {
+			switch platformName {
+				case "iPhoneOS", "iPhoneSimulator":
+					return .iOS
+				case "WatchOS", "WatchSimulator":
+					return .watchOS
+				case "AppleTVOS", "AppleTVSimulator":
+					return .tvOS
+				default:
```

Comment:
What about `VisionOS`? 😛

Reply:
Going to pull that change into a separate branch, there's a few more files to change.